config/storybook/config.js

```diff
@@ -0,0 +1,22 @@
+import { configure } from '@storybook/react';
+
+const reqs = [
```

Comment:
I think a comment linking to the explanation of this in `config/storybook/webpack.config.js` would be helpful.

Reply:
Done 👍